netlabel/labels.go

```diff
@@ -86,3 +92,23 @@ func Value(label string) string {
 
 	return kv[1]
 }
+
+// KeyValue decomposes the label in the (key,value) pair
+func KeyValue(label string) (string, string, error) {
```

Comment:
I could not find a docker package dedicated to label parsing only. If there is a simple library dedicated to that I would use it.
Label parsing/handling is so simple that having your own few lines implementation should not be a problem, IMO.